PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,3 @@
-# [DEV-]()
```

Comment:
Hmm it's worked for me without linking the ticket ID in the description, see this PR https://github.com/sparklayer-io/spark-api-theme/pull/44

Perhaps we update it to `# DEV-`

Reply:
Sounds good, lets try it!

Reply:
@jam3sn hmm tried it here on this ticket and it did not work: https://github.com/sparklayer-io/spark-js/pull/1226

Reply:
Must be a larger issue with the integration then :/